README.md

```diff
@@ -13,6 +13,25 @@ This will install all the dependencies for the project.
 Runs the app in the development mode.\
 Open [http://localhost:3001](http://localhost:3001) to view it in the browser.
 
+### Configure custom substrate providers
+
+In the `.env` file at root project directory, you have to define the following env variables:
+
+REACT_APP_SUBSTRATE_PROVIDER_1=<WS-Provider chain 1>
+REACT_APP_SUBSTRATE_PROVIDER_2=<WS-Provider chain 2>
+REACT_APP_PROVIDER_NAME_1=<Chain 1 name>
+REACT_APP_PROVIDER_NAME_2=<Chain 2 name>
+
+# In case these values are not provided the default chains will be rialto and millau chains.
+
+### `yarn test`
+
+Launches the test runner in the interactive watch mode.\
+See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+
+### Configure custom substrate providers
+
+In the `.env` file at root project directory, you
```

Comment:
```suggestion
```

Reply:
why do you want to remove this?

Reply:
It's duplicated here, two paragraphs up you have exactly the copy and this part is simply unfinished.